docs/modules/cluster-performance/pages/thread-per-core-tpc.adoc

```diff
@@ -0,0 +1,97 @@
+= Thread-Per-Core (TPC)
+:description: Find out more about Thread-Per-Core (TPC) and how to enable this feature on clients and cluster members.
+:page-enterprise: true
+
+Enable Thread-Per-Core (TPC) on both clients and cluster members to improve system performance, especially on machines with either fewer or more CPU cores than the recommended hardware listed on the xref:ROOT:production-checklist.adoc[].  
```

Comment:
```suggestion
Enable Thread-Per-Core (TPC) on both clients and cluster members to improve system performance. This is especially effective if the number of CPU cores on your machines does not match the recommended hardware in the xref:ROOT:production-checklist.adoc[].  
```

Reply:
Actually, production checklist states 'at least 8' so we do need to revert this or clarify the overall recommendation...